src/sentry/relay/config/metric_extraction.py

```diff
@@ -825,11 +825,15 @@ def _generate_span_attribute_specs(project: Project) -> list[HashedMetricSpec]:
     ):
         return []
 
-    extraction_rules_state = MetricsExtractionRuleState.load_from_project(project)
+    extraction_configs = SpanAttributeExtractionRuleConfig.objects.filter(project=project).all()
```

Comment:
`all()` is not needed, usually, `all()` is used only if you want to select all object without using the filter method.
e.g.:
```
SpanAttributeExtractionRuleConfig.objects,all()
```